src/__tests__/lib/containers/GenericCard.test.tsx

```diff
@@ -108,6 +108,11 @@ describe('it renders the UI correctly', () => {
     expect(wrapper.find(Utils.CardFooter)).toHaveLength(1)
   })
 
+  it.skip('renders the query handles corretly', () => {
```

Comment:
need a test for validating data attributes, enzyme isn't rendering data attributes though